src/autocomplete/EmojiProvider.tsx

```diff
@@ -69,15 +73,15 @@ export default class EmojiProvider extends AutocompleteProvider {
 
     constructor() {
         super(EMOJI_REGEX);
-        this.matcher = new QueryMatcher<IEmojiShort>(EMOJI_SHORTNAMES, {
-            keys: ['emoji.emoticon', 'shortname'],
+        this.matcher = new QueryMatcher<IEmojiShort>(EMOJI_SHORTCODES, {
+            keys: ['emoji.emoticon', 'shortcode'],
```

Comment:
```suggestion
            keys: ['emoji.emoticon', 'shortcodes'],
```

QueryMatcher now supports indexing on array fields, this can replace the funcs bit below once you merge shortcode+altShortcodes

Reply:
If we zip the data then it'll likely want to be `funcs: [e => `:${e.emoji.shortcodes}:`],` or similar

Reply:
> supports indexing on array fields

Oh, nice